slack/endpoint.bal

```diff
@@ -14,6 +14,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerinax/'client.config;
 import ballerina/http;
```

Comment:
Let's exchange these imports to follow the standard of importing libraries in alphabetic order. Apply where necessary.

Reply:
+1